extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -176,9 +198,9 @@ export class PodcastPlayer extends Component {
 			<section
 				className={ cssClassesName }
 				style={ Object.keys( inlineStyle ).length ? inlineStyle : null }
-				aria-labelledby={ title || ( track && track.title ) ? `${ playerId }__title` : undefined }
+				aria-labelledby={ title || ( track && track.title ) ? playerId + '__title' : undefined }
 				aria-describedby={
-					track && track.description ? `${ playerId }__track-description` : undefined
+					track && track.description ? playerId + '__track-description' : undefined
```

Comment:
Were the template strings causing issues when linting, or was there another reason to get rid of them there? I am asking because it seems that our most recent eslint update in the project introduced some issues when linting template strings.

If possible, I think it would be nice to use template strings instead of concatenation when possible, to keep some consistency.

Reply:
Yes, the template strings were causing linting issues, so I was unable to commit without utilizing `--no-verify`. These are not lines I actually needed to change for my PR, other than to be able to commit without `--no-verify` 😞

Reply:
Was able to revert back to the template literal after utilizing this patch: https://github.com/Automattic/jetpack/pull/15241